cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -245,6 +245,18 @@ private HiveConf getHiveConf() {
       conf.set(HIVE_METASTORE_TOKEN_KEY, HiveAuthFactory.HS2_CLIENT_TOKEN);
     }
 
+    // workaround to allow CDAP explore to modify params of session conf
+    String whiteListAppend = conf.getVar(HiveConf.ConfVars.HIVE_AUTHORIZATION_SQL_STD_AUTH_CONFIG_WHITELIST_APPEND);
+    String paramsExploreModifies =
+      "spark.*|explore.*|mapreduce.*|hive.*|" + CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION;
```

Comment:
I left `spark.*` (because we set all the hadoop configs prefixed by 'spark.') and `explore.*` (because we set a prefix a lot of params with 'explore.').
I removed the star from mapreduce/hive params.